extensions/interactions/customization-args-defs.ts

```diff
@@ -283,28 +283,28 @@ export interface MathEquationInputCustomizationArgs { }
 
 
 export interface MultipleChoiceInputCustomizationArgsBackendDict {
-  showChoicesInShuffledOrder?: {
-    value: string;
+  showChoicesInShuffledOrder: {
+    value: boolean;
```

Comment:
Wow good catch @nishantwrp. Why wasn't this detected before?

Reply:
It is because there is no usage of this interface that imply `showChoicesInShuffledOrder` is a `bool`. It was used in one place as 

```typescript

if (showChoicesInShuffledOrder) {
...
}

```

Now as you can see it's also valid if it were a string.